test/libraries/Eip712.sol

```diff
@@ -4,6 +4,13 @@ pragma solidity ^0.8;
 import {GPv2Order} from "src/contracts/libraries/GPv2Order.sol";
 
 library Eip712 {
+    struct EIP712Domain {
+        string name;
+        string version;
+        uint256 chainId;
+        address verifyingContract;
+    }
```

Comment:
nit: per https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator suggest that there be a comment that `salt` is intentionally omitted from the domain separator.